modules/openapi-generator/src/main/resources/typescript-node/api-single.mustache

```diff
@@ -1,6 +1,13 @@
 {{>licenseInfo}}
+
+{{#supportsES6}}
+import localVarRequest from 'request';
+import http from 'http';
+{{/supportsES6}}
+{{^supportsES6}}
 import localVarRequest = require('request');
```

Comment:
The import var = require does not make much sense to me. This should be probably `const localVarRequest = require('request')`.

Or did I oversee something?

Reply:
Same for the other parts.

Reply:
Yeah thats right, i'll fix it :)

Reply:
I'm getting some typescript errors with const http = require. These are "TS2503: Cannot find namespace 'http'." and "error TS4055: Return type of public method from exported class has or is using private name 'http'.". As described here: https://stackoverflow.com/questions/31173738/typescript-getting-error-ts2304-cannot-find-name-require there are some ways to make this work, but to be honest i don't see a reason why to make a difference at this point. All other modules are imported with the "import .. from" stmt independent of the ES6 flag. So i removed the switch and changed the tsconfig to make the imports http/request work. What do you think about this solution?

Reply:
For TypeScript to use these you need the types for the Nodejs internal ones. Basically `@types/node`.

Reply:
The current solution should be fine.